bot_commands.py

```diff
@@ -0,0 +1,128 @@
+import json
+
+import requests
+from sqlalchemy import func
+
+from credentials import bot_token
+from init import db
+from models import User, Topic
+from speeches import speeches
+
+TOKEN = bot_token
+
+
+def send_message(data, method='sendMessage'):
+    url = f"https://api.telegram.org/bot{TOKEN}/{method}"
+    requests.post(url, data=data)
+
+
+def parse_callback_query(request):
+    data = {"chat_id": request["message"]["chat"]["id"],
+            "parse_mode": "Markdown",
+            "disable_web_page_preview": "true"}
+    topic_name = request["data"]
+    topic = Topic.query.filter_by(name=topic_name).first()
+    text = f"I want to advise you these materials for learning *{topic_name}*:\n\n"
+    if not topic:
+        return
+    text += "-"*40 + "\n"
```

Comment:
Probably it could be written in a more short way (this text forming) but it's up to you